apps/system-e2e/src/tests/islandis/application-system/acceptance/parental-leave.spec.ts

```diff
@@ -16,13 +20,51 @@ import {
   EmailAccount,
   makeEmailAccount,
 } from '../../../../support/email-account'
-import { helpers, locatorByRole } from '../../../../support/locator-helpers'
+import { helpers } from '../../../../support/locator-helpers'
 import { session } from '../../../../support/session'
-import { mockApi } from '../../../../support/api-tools'
 import { setupXroadMocks } from './setup-xroad.mocks'
 
 test.use({ baseURL: urls.islandisBaseUrl })
 
+async function getEmployerEmailAndApprove(employer: EmailAccount, page: Page) {
+  const { proceed } = helpers(page)
+
+  const email = await employer.getLastEmail(6)
+
+  if (email && typeof email.html === 'string') {
```

Comment:
Could be reordered to use guard-pattern; reducing indentation
```suggestion
  // Require email
  if (!email || typeof email.html !== 'string') { throw new Error('Email not found, test incomplete') }
```